sample/Sample/Program.cs

```diff
@@ -20,7 +22,7 @@
 
     foreach (var i in Enumerable.Range(0, 100))
     {
-        Log.Information("Running loop {Counter}, switch is at {Level}", i, levelSwitch.MinimumLevel);
+        Log.Information("Running loop {Counter.I}, switch is at {Level}", i, levelSwitch.MinimumLevel);
```

Comment:
Should the example just be left as-is? I can't imagine actually wanting to log a thing called _Counter.I_

Reply:
Oops, thanks! Fixed now :+1: